numpy/core/tests/test_scalarmath.py

```diff
@@ -898,7 +898,8 @@ def test_scalar_integer_operation_overflow(dtype, operation):
 @pytest.mark.parametrize("operation", [
         lambda min, neg_1: abs(min),
         lambda min, neg_1: min * neg_1,
-        lambda min, neg_1: min // neg_1], ids=["abs", "*", "//"])
+        pytest.param(lambda min, neg_1: min // neg_1,
+            marks=pytest.mark.xfail)], ids=["abs", "*", "//"])
```

Comment:
Probably need to comment it out or skip this, because it is just too broken...

Reply:
At the risk of slightly expanding the scope of this PR, I went ahead and just avoided the actual error-causing computation in the integer division function, setting it to return `NPY_MIN_@NAME@` as I think matches `gcc` behavior (in lieu of true defined behavior, this is the next-best thing imo). The test runs now, and the behavior matches expectations I think.

Reply:
OK, I would like to backport these division fixes at some point.  But I am starting to suspect we should make a PR with those from scratch… (i.e. a backport only PR)